modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.soy

```diff
@@ -142,7 +144,23 @@
 						</div>
 
 						<div class="list-group-item-field">
+							{let $titleMessage kind="text"}
+								{msg desc=""}due-to-missing-fields{/msg}
+							{/let}
+							{let $labelMessage kind="text"}
+								{msg desc=""}broken-rule{/msg}
+							{/let}
```

Comment:
@alinedoleron, these variables should be declared inside the `if`, once they'll only be used if the rule is invalid.